lib/instances/Game.lua

```diff
@@ -8,8 +8,15 @@ function Game:init(instance)
 
 	Instance.new("RunService", instance)
 	Instance.new("CoreGui", instance)
+	Instance.new("GuiService", instance)
+	Instance.new("HttpService", instance)
+	Instance.new("LocalizationService", instance)
+	Instance.new("Players", instance)
 	Instance.new("ReplicatedStorage", instance)
 	Instance.new("TestService", instance)
+	Instance.new("TextService", instance)
+	Instance.new("TweenService", instance)
+	Instance.new("UserInputService", instance)
```

Comment:
Extremely minor nitpick, UserInputService is, for whatever reason, not a child of game, and is always needed to received through GetService. I'm not sure if we care enough to replicate this behavior.

Reply:
That's super weird. I had no idea you could even _do_ that?

I wonder if we can just fix that in Roblox instead of making Lemur emulate that, lol.

Reply:
...What? Hahaha. Yeah that'd be cool to fix, it's annoying!

Reply:
I did `print(game.Instance.ClassName)` earlier, says it's KeyframeSequenceProvider, or something. Seems like its happening to much more services.